src/python/ensembl/io/genomio/database/meta_getter.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A simple helper script to connect to a core database and retrieve a single meta_value 
+or multiple meta_value and dump meta_key/value pairs to stdout / JSON."""
+
+__all__ = ["get_meta_values"]
+
+import argparse
+import logging
+import json
+from pathlib import PosixPath
+from pathlib import Path
+
+from sqlalchemy.engine import URL
+
+from ensembl.utils.argparse import ArgumentParser
+from ensembl.utils import StrPath
+from ensembl.utils.logging import init_logging_with_args
+from .dbconnection_lite import DBConnectionLite
+
+
+def get_meta_values(server_url: URL, db_name: str, input_queries: StrPath) -> dict[str, str] | None:
```

Comment:
```suggestion
def get_meta_values(server_url: URL, db_name: str, meta_keys: StrPath | list[str]) -> dict[str, str]:
```
I find `input_queries` not too related with what this variable contains.